### PR TITLE
feat(webvitals): Updates webvitals module tables to use stored scores

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Updates the following Webvitals modules components to use backend stored scores
- Landing page table
- Landing page Webvital slideout table
- Page Overview table
- Page Overview Webvital slideout table

Also updates Landing page table and Page Overview table to be sortable by performance score and opportunity score

### CODE REVIEW

File: static/app/views/performance/browser/webVitals/utils/useStoredScoresSetting.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+export function useStoredScoresSetting() {
+  const location = useLocation();
+  const {query} = location;
+
+  if (query.useStoredScores !== undefined) {
+    return decodeScalar(query.useStoredScores) === 'true';
+  }
+
+  return USE_STORED_SCORES;
```
Comment (k-fish) [inline static/app/views/performance/browser/webVitals/utils/useStoredScoresSetting.tsx:13]: 👍

File: static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/useTransactionSamplesWebVitalsScoresQuery.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                      (toNumber(row[`measurements.score.weight.${webVital}`]) ?? 0)) *
+                      100
+                  ),
+                  [`${webVital}Weight`]: Math.round(
+                    (toNumber(row[`measurements.score.weight.${webVital}`]) ?? 0) * 100
+                  ),
+                }
+              : {}),
+          })
+          // TODO: Discover doesn't let us query more than 20 fields and we're hitting that limit.
```
Comment (k-fish) [inline static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/useTransactionSamplesWebVitalsScoresQuery.tsx:125]: Or we can change that limit 🤷, is that a pre-metrics or metrics constraint?

**Review by k-fish — APPROVED** [review]
It's behind the const, so let's try it out 👍

**Review by gggritso — APPROVED** [review]
👍🏻

File: static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/useTransactionSamplesWebVitalsScoresQuery.tsx (hunk 2 above)
Comment (edwardgou-sentry) [reply static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/useTransactionSamplesWebVitalsScoresQuery.tsx:125]: I _think_ it's an artificial constraint in discover to proactively reduce load to clickhouse.
I don't actually need to query that many fields for now, but yeah we can look into changing the limit if it becomes a blocker.

**Pull request merged**